include/util.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_MATH_H_
+#define ZEPHYR_INCLUDE_MATH_H_
+
+/** Return the maximum of a and b */
+#define max(a, b)	\
```

Comment:
did you check MIN and MAX in include/sys/util.h?

Reply:
:-) OK I need to check my ctags settings. I thought it was odd.